example/cavity/Fluid/constant/transportProperties

```diff
@@ -1,18 +1,24 @@
 /*--------------------------------*- C++ -*----------------------------------*\
 | =========                 |                                                 |
-| \\      /  F ield         | foam-extend: Open Source CFD                    |
-|  \\    /   O peration     | Version:     3.2                                |
-|   \\  /    A nd           | Web:         http://www.foam-extend.org         |
+| \\      /  F ield         | OpenFOAM: The Open Source CFD Toolbox           |
+|  \\    /   O peration     | Version:  v1912                                 |
+|   \\  /    A nd           | Website:  www.openfoam.com                      |
 |    \\/     M anipulation  |                                                 |
 \*---------------------------------------------------------------------------*/
 FoamFile
 {
     version     2.0;
     format      ascii;
     class       dictionary;
-    object      turbulenceProperties;
+    location    "constant";
+    object      transportProperties;
 }
 // * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * //
 
-simulationType laminar;
+transportModel  Newtonian;
 
+nu      nu      [0 2 -1 0 0 0 0]        0.01;
+pRef    pRef    [1 -1 -2 0 0 0 0]       0.0;
+rho     rho     [1 -3 0 0 0 0 0]        1.0;
```

Comment:
I think that pimpleFoam only needs `nu`, the rest are default. See an example: https://github.com/precice/tutorials/blob/master/partitioned-pipe/fluid1-openfoam-pimplefoam/constant/transportProperties